### PR TITLE
chore(deps): update all dependencies

### PR DESCRIPTION
**Pull request by masone**
will issue a breaking change just to be sure...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,32 +18,32 @@
...
-    "eslint-plugin-react": "^7.20.0",
-    "eslint-plugin-react-hooks": "^4.0.2",
+    "eslint-plugin-jest": "^24.0.1",
+    "eslint-plugin-jsdoc": "^30.5.1",
+    "eslint-plugin-json": "^2.1.2",
+    "eslint-plugin-prefer-arrow": "^1.2.2",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.20.6",
+    "eslint-plugin-react-hooks": "^4.1.2",
     "eslint-plugin-testing-library": "^3.1.2",
```
Comment (masone) [inline package.json:35]: need to keep this one down because of a peer dependency conflict introduced in later versions

**Review by Averethel — APPROVED** [review]

**Comment by Averethel** [comment]
@masone anything I can do to help here?

**Comment by masone** [comment]
I'm not on top of this, feel free to take it, but I don't feel it's pressing. 

I think we have some dependency updates in flight that are hard to deal with (dependencies of dependencies, conflicting peers, etc)

**Comment by Averethel** [comment]
I checked in projects, there does not seem to be necessary to issue a major update, bugfix should be fine

**Pull request merged**